apps/src/lab2/views/components/starterAssetsDialog/StarterAssetsDialog.tsx

```diff
@@ -64,22 +66,25 @@ const StarterAssetsDialog: React.FC<SelectProps | UploadProps> = props => {
     [assets]
   );
 
-  const setError = useCallback(
+  const handleError = useCallback(
     (error: Error) => {
       onError?.(error);
-      setShowError(true);
+      setCurrentError(error);
```

Comment:
nit/optional: might be cleaner to go ahead and compute the error message string, and just store the error message string instead of passing the `error` object down into `ErrorAlert`? That way `ErrorAlert` can just accept a string `errorMessage`.

Reply:
Good call, updated!